CHANGELOG.md

```diff
@@ -1,5 +1,91 @@
 # [Versions](https://mui.com/versions/)
 
+## v5.15.5
+
+<!-- generated comparing v5.15.4..master -->
+
+_Jan 17, 2024_
+
+A big thanks to the 18 contributors who made this release possible. Here are some highlights ✨:
+
+- Base UI's CSS class prefix is now `base-` (#40205) @michaldudak
+- Bug fixes, and a lot of improvements to code and docs infra
+
+### `@mui/material@5.15.5`
+
+- &#8203;<!-- 42 -->[Accordion] Deprecate \*Props props and classes for v6 (#40418) @DiegoAndai
+- &#8203;<!-- 41 -->[material-ui][Alert] Update TypeScript types to allow color override types to be added to `iconMapping` and `severity` props (#40551) @2metres
+- &#8203;<!-- 27 -->[Dialog] Remove deprecated onBackdropClick from Dialog tests (#40505) @sai6855
+- &#8203;<!-- 26 -->[Divider] Deprecate props and classes for v6 (#40563) @sai6855
+
+### `@mui/material-next@6.0.0-alpha.118`
+
+- &#8203;<!-- 36 -->[ButtonGroup] Apply MD3 style to `ButtonGroup` (#40124) @lhilgert9
+
+### `@mui/base@5.0.0-beta.32`
+
+- &#8203;<!-- 40 -->Change the CSS class prefix to `base-` (#40205) @michaldudak
```

Comment:
Do we need to include details about the breaking change - https://github.com/mui/material-ui/pull/40205#issue-2040482981?

Reply:
👍 Good idea ~